pages/register.js

```diff
@@ -0,0 +1,245 @@
+import React from 'react';
+import { Formik, Field, FieldArray, Form, ErrorMessage } from 'formik';
+import superagent from 'superagent';
+
+import { businessDays } from '../utils/businessDays';
+import { businessHours } from '../utils/businessHours';
+import { instruments } from '../utils/instruments';
+
+import validateEmail from '../utils/validateEmail';
+import validateZipCode from '../utils/validateZipCode';
+import validatePhone from '../utils/validatePhone';
+
+export default function RegisterPage() {
+  async function handleSubmit(values) {
+    console.log(values);
+    // const response = await superagent.post(
+    //   'http://localhost:3000/api/register',
+    //   JSON.stringify(values)
+    // );
+    // console.log(response);
+  }
+
+  return (
+    <div>
+      <Formik onSubmit={handleSubmit}>
+        {({
+          values,
```

Comment:
I f eel I made comments about this before? Let's move all these values outside of the return to have cleaner code

Reply:
Ignore this. It was resolved in other PR!